src/main/resources/db/changelog/changesets/changelog_20230821T072105Z.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-4.1.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.1.xsd">
+    <changeSet author="yenguised (generated)" id="1692602478163-1">
+        <addColumn tableName="sensitivity">
+            <column name="contingency_id" type="varchar(255)"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="yenguised (generated)" id="1692602478163-2">
+        <addColumn tableName="sensitivity">
+            <column name="contingency_type" type="varchar(255)"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="yenguised (generated)" id="1692602478163-3">
+        <addColumn tableName="sensitivity">
+            <column name="function_id" type="varchar(255)"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="yenguised (generated)" id="1692602478163-4">
+        <addColumn tableName="sensitivity">
+            <column name="function_reference_after" type="float8"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="yenguised (generated)" id="1692602478163-5">
+        <addColumn tableName="sensitivity">
+            <column name="function_type" type="varchar(255)"/>
+        </addColumn>
+    </changeSet>
+    <changeSet author="yenguised (generated)" id="1692602478163-6">
+        <addColumn tableName="sensitivity">
+            <column name="id" type="varchar(255)"/>
```

Comment:
Why is there 'id' and 'sensitivityId' ? Column id is empty in database ?

Reply:
the id is for contingency, not the sensitivity. i change it to contingencyId to make it clear